plugins/flowdock/app/decorators/stage_decorator.rb

```diff
@@ -0,0 +1,21 @@
+Stage.class_eval do
```

Comment:
yeah true, will make the plugin fail nicely :)

FYI alternative approach we used before to get constants:

```
Stage
class Stage
   ... use constants ...
end
```